tests/bluetooth/ctrl_sw_privacy_unit/src/main.c

```diff
@@ -80,19 +80,25 @@ void helper_privacy_add(int skew)
 	zassert_equal(pos, FILTER_IDX_NONE, "");
 }
 
-void test_privacy_clear(void)
+static void setup(void *data)
 {
+	/* Clear the cache so we start fresh each time. */
 	prpa_cache_clear();
+}
 
+ZTEST_SUITE(test_ctrl_sw_privacy_unit, NULL, NULL, setup, NULL, NULL);
```

Comment:
`setup` is being passed to the `before` field of the suite macro, either rename or relocate please